cms/__init__.py

```diff
@@ -60,21 +60,28 @@
 LANG_CPP = "cpp"
 LANG_CS = "cs"
 LANG_PASCAL = "pas"
-LANG_PYTHON = "py"
+LANG_PYTHON2 = "py2"
+LANG_PYTHON3 = "py3"
 LANG_PHP = "php"
 LANG_JAVA = "java"
 
+# Python 3 py_compile outputs to __pycache__/filename.<PYTHON3_COMPILE_NAME>.pyc
+# Using a '*' in this place does not appear to work correctly.
+# Format seems standard as cpython-<MAJOR><MINOR> for cpython.
+PYTHON3_COMPILE_NAME = "cpython-%d%d" % (3, 4)
```

Comment:
Sure, that makes sense, if we're going to that effort though, would it make more sense to just do an `os.listdir('__pychache__')` to get the actual file name?
The advantage would be that it's a) local to the part of the code that cares and b) if a non-cpython implementation was used it would also deal with that, unlike hardcoding 'cpython' into the name.
The disadvantage is that it would call `os.listdir` every time a python3 program is compiled.